repos/keg-components/src/components/touchable/touchable.native.js

```diff
@@ -7,10 +7,13 @@ import {
   TouchableWithoutFeedback,
 } from 'react-native'
 import { useClassName } from 'KegClassName'
+import { useClassOnPressIn } from 'KegClassOnPressIn'
```

Comment:
marking this file for reference.
For any other component that uses `Touchable` directly, a similar bug still occurs, though it's slightly different than how it occured before.

* it sometimes removes the classNames on `keg-touchable` 
* it happens occasionally even if you don't click + drag

on `keg-components:` 
* `textToggle` component
    * [gif](https://user-images.githubusercontent.com/3317835/100653453-aa4da380-3305-11eb-9c92-825c7aa9f9a6.gif)

* `Touchable` component
    * [gif](https://user-images.githubusercontent.com/3317835/100653200-49be6680-3305-11eb-8470-44f1be64f361.gif)

on `tap evf`:
* Mobile view - expandable session
   * the grid item is wrapped with `Touchable`
   * when you click + drag to the side, the styles on the `Touchable` component dissapears
   * [gif](https://user-images.githubusercontent.com/3317835/100654021-77f07600-3306-11eb-9279-3aca34291666.gif)
* Session Links
   * click + drag still occurs, but unlike before, i'm able to bring the styles back if I click on the link again
   * [gif](https://user-images.githubusercontent.com/3317835/100654323-d9184980-3306-11eb-8dbc-49f0b02bc6f2.gif)